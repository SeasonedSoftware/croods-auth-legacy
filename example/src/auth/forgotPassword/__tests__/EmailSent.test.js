import React from 'react'
import renderer from 'react-test-renderer'

import EmailSent from '../EmailSent'

it('renders correctly', () => {
  const tree = renderer.create(<EmailSent />).toJSON()
  expect(tree).toMatchSnapshot()
})
