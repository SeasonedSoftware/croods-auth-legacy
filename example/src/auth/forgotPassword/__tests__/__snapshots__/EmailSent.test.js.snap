// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "justifyContent": "center",
      "marginTop": "20vh",
      "width": "100%",
    }
  }
>
  <h2>
    We sent you an email with instructions to reset your password
  </h2>
</div>
`;
